docs/mosviz/import_data.rst

```diff
@@ -12,8 +12,23 @@ Automatic Directory Loading
 Mosviz provides instrument-specific directory parsers for select instruments. At this
 time, Mosviz supports automatic parsing for the following instruments:
 
-* JWST NIRSpec
+* JWST NIRSpec (levels 2 and 3)
 * JWST NIRISS
+* JWST NIRCam
+
+The NIRSpec parser expects a directory with either level 2 files:
+
+* ``*_cal.fits`` : Single file containing level 2 2D spectra for all objects.
+* ``*_x1d`` : Single file containing level 2 1D spectra for all objects.
+
+or level 3 files:
+
+* ``*_s2d.fits`` : N files containing level 3 2D spectra, where N is the number of objects.
+* ``*_x2d.fits`` : N files containing level 3 1D spectra, where N is the number of objects.
```

Comment:
I don't think ``x2d`` exists. I cannot find it in https://jwst-pipeline.readthedocs.io/en/stable/jwst/data_products/product_types.html#stage-3-data-products

```suggestion
* ``*_x1d.fits`` : N files containing level 3 1D spectra, where N is the number of objects.
```